tests/v2/test_waku_swap.nim

```diff
@@ -142,3 +142,108 @@ procSuite "Waku SWAP Accounting":
       node2.wakuSwap.accounting[node1.peerInfo.peerId] == -1
     await node1.stop()
     await node2.stop()
+
+
+
+  asyncTest "Add peer to blacklist after disconnect Threshold is reached":
+    let
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node1 = WakuNode.new(nodeKey1, ValidIpAddress.init("0.0.0.0"),
+        Port(60000))
+      nodeKey2 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node2 = WakuNode.new(nodeKey2, ValidIpAddress.init("0.0.0.0"),
+        Port(60001))
+      contentTopic = ContentTopic("/waku/2/default-content/proto")
+      message = WakuMessage(payload: "hello world".toBytes(), contentTopic: contentTopic)
+
+    var completionFut = newFuture[bool]()
+    var futures = [newFuture[bool](), newFuture[bool]()]
+
+
+    # Define the waku swap Config for this test
+    let swapConfig = SwapConfig(mode: SwapMode.Mock, paymentThreshold: 1, disconnectThreshold: -1, stateUpdateOverRide: true, closePeerConnection: false)
```

Comment:
Ditto above, I'm confused about this SwapConfig considering the test comment "Add peer to blacklist after disconnect Threshold is reached"